streaming/src/main/scala/org/apache/spark/streaming/scheduler/RateController.scala

```diff
@@ -86,5 +86,5 @@ private[streaming] abstract class RateController(val streamUID: Int, rateEstimat
 
 object RateController {
   def isBackPressureEnabled(conf: SparkConf): Boolean =
-    conf.getBoolean("spark.streaming.backpressure.enable", false)
+    conf.getBoolean("spark.streaming.backpressure.enabled", false)
```

Comment:
The idea here was that this flag hadn't been 'released' yet so OK to change the name without keeping the old one?

The two changes in the PR were pretty unrelated right? still I can't imagine someone would really care about back-porting one and not be able to consume the other as they're minor.